src/bio/smd/smd_pool.c

```diff
@@ -300,40 +327,57 @@ int
 smd_pool_replace_blobs_locked(struct smd_pool_info *info, int tgt_cnt,
 			      uint32_t *tgts)
```

Comment:
As I mentioned above, let's assume smd_pool_replace_blobs_locked() supports DATA device only for this moment.

Reply:
Called from smd_dev_replace(), I think we should add a device type parameter

Reply:
Let's just revert the change and assume only DATA device is supported for this moment. To support META/WAL devices, I think we need to make a detailed design first, but let's focus on basic I/O functionalities for this moment.

Reply:
Done